frontend/src/container/FeedItemWithPreview.jsx

```diff
@@ -411,7 +411,7 @@ export class FeedItemWithPreview extends React.Component {
                 translationTargetLanguageList={props.system.config.translation_service__target_languages}
                 workspaceId={props.workspaceId}
                 // End of required props ///////////////////////////////////////
-                codeLanguageList={props.system.config.code_languages}
+                codeLanguageList={props.system.config.ui__notes__code_sample_languages}
```

Comment:
if "__" (the 2 underscores) acts like a separator, wouldn't it be better to be ui.notes.code_sample_languages?

Reply:
``ui__notes__code_sample_languages`` is a variable send from the backend. 
I didn't change anything (see [here before my mention development](https://github.com/tracim/tracim/blob/2a2eda36027a96e84a782af06fd2a1b5a55adeda/frontend/src/util/tinymceInit.js#L103)). I'm agree that it's not the best naming, however this PR is to fix the regressions... 😕

Reply:
ok, don't change it then

Reply:
you can resolve